app/assets/config/manifest.js

```diff
@@ -0,0 +1,21 @@
+//= link_tree ../images
```

Comment:
This is super weird, putting images, css and fonts all into a `.js` manifest. 

Is the api in sprockets right? should there be multiple manifest files?